apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -34,6 +36,17 @@ export class WorkspaceController {
     this.sessionService.setCwd(path);
   }
 
+  @Post('create')
+  create(@Body() body: WorkspaceCreateDto): ExecResult {
```

Comment:
on one hand, it makes sense to put it here. However, `ng new` is part of generators, from the point of view of the CLI. Following my yesterday conversation with @iYadgar and Oryan Baruch (sorry, don't know GitHub nick), maybe it won't be a good idea to split it and place it here.

Reply:
Ok I'll move it to the generators.service but than we're creating a dependency between workspace and generators which is fine but something to be aware of